src/schema/artist/index.js

```diff
@@ -243,14 +264,27 @@ export const ArtistType = new GraphQLObjectType({
           }
           return auctionLotLoader(diffusionArgs).then(
             ({ total_count, _embedded }) => {
-              return assign(
+              const totalPages = Math.ceil(total_count / size)
+              return merge(
                 {
-                  pageCursors: createPageCursors({ page, size }, total_count),
+                  pageCursors: createPageCursors(
+                    {
+                      page,
+                      size,
+                    },
+                    total_count
+                  ),
                 },
                 connectionFromArraySlice(_embedded.items, options, {
                   arrayLength: total_count,
                   sliceStart: offset,
-                })
+                }),
+                {
+                  pageInfo: {
+                    hasPreviousPage: page > 1,
+                    hasNextPage: page < totalPages,
+                  },
+                }
```

Comment:
I think at some point we could wrap line 221-287 in a util helper since all (or rather, most) of these items appear to be generic and reusable, but for now nice and straight forward 👍

Reply:
Yea, I'm going to be adding this, merging in our own `hasPage` methods into a couple of other connection types soon (related artists, etc.). At that point can maybe refactor into some wrapper.